postgres/postgres-entrypoint.sh

```diff
@@ -43,4 +43,4 @@ if [ "$1" = 'postgres' ]; then
     cdc_setup_hba_conf
     bind_wal2json
 fi
-exec /docker-entrypoint.sh "$@"
+exec docker-entrypoint.sh "$@"
```

Comment:
```suggestion
exec /usr/local/bin/docker-entrypoint.sh "$@"
```

or use the full path to be explicit?